### ISSUE TITLE
Multiplayer Signalling Disconnect Don't disconect the Host

### ISSUE THREAD

**Issue by kingpirux** [Scirra/Construct-3-bugs#1709]
## Problem description

If the host is disconnected from the Signalling server without closing the room first, the room persist in the system and can't create or login to other rooms again.

## Attach a .c3p

https://drive.google.com/open?id=1Y1aO2So_2RFBeCbpeMJY0SkRw9-2qyuA

## Steps to reproduce

1. Run Login Layout
2. Login with desired alias
3. Click DisconnectSignalling
4. Click ConnectSignalling
5. Click CreateRoom

## Observed result

If disconnected without closgin the room first, the user is disconnected from signalling, but the system persist that it is logged in and hosting a room so the peer can't connect or disconnect.

## Expected result

The room is closed, the host is disconnected and the user can login again to create or join to a new room.

## Affected browsers

* Chrome: yes

## System details
Construct 3 r105

There is no error dialog, no explorer console error, no multiplayer plugin error warning.

**Comment by AshleyScirra** [comment]
This is by design. There are two connections once you've joined a room: the connection to the signalling server, and the connection to the host (or connections to the peer if you are the host). If you disconnect from the signalling server, it doesn't necessarily disconnect your host/peer connections - in other words you remain in the room. If you then reconnect to the signalling server you're still in the room, so you can't rejoin another room until you leave the current room you're in.

In other words, the signalling connection and the peer connection are two separate things which operate independently.
